docs/get-started/how-it-works.md

```diff
@@ -1,23 +1,24 @@
-# How Does it Work?
+# {octicon}`question` How Does it Work?
 
-## Contrastive metric learning
+Finetuner is a framework to improve models that encode data into embedding for similarity matching.
+From an algorithmic perspective, it leverages the contrastive learning approach.
+This involves three steps:
 
-From an algorithmic perspective,
-**Finetuner** leverages a contrastive metric learning approach to improve your model.
-How does it work?
+## Step 1: Build embedding model
 
-### Step 1: Convert a model into an embedding model
+Finetuner interprets the architecture of an existing (pre-trained model) which we call backbone.
+Those model might not be an embedding model upfront and the architecture not suitable for training it to encode data into embeddings.
+Therefore, Finetuner removes the default *head*, applies *pooling* and freezes layers that do not need to be trained.
```

Comment:
```suggestion
Finetuner takes an existing, pre-trained model, typically called the __backbone__, and analyzes its architecture. If this model does not already produce embeddings, Finetuner removes the default *head* (the upper layers of the network), applies *pooling*, and freezes any layers that do not need to be trained.
```

Reply:
The model might produce embeddings before, but they might not be good, also freezing is optional. Therefore, I will stick to the old formulation